src/NupkgWrench/NuspecCommands/DependenciesUtil.cs

```diff
@@ -74,8 +73,18 @@ public static void Process(XDocument nuspecXml, EditType verb, HashSet<NuGetFram
                     }
                 }
             }
+            else
+            {
+                // if no framework, working on dependencies node
+                if (groups.Count == 0)
```

Comment:
The nuspec may not contain a dedpendencies node at all, in which case there is nothing to remove/modify. It looks like with this change you would get a null exception later, so you'll need to a check here
`if (groups.Count == 0 && dependenciesNode != null)`